src/jarabe/desktop/keydialog.py

```diff
@@ -102,10 +102,19 @@ def __init__(self, ssid, flags, wpa_flags, rsn_flags, dev_caps, response):
 
     def add_key_entry(self):
         self._entry = Gtk.Entry()
+        self._entry.set_visibility(False)
```

Comment:
You can just pass properties as constructor kwargs, eg:

    self._entry  = Gtk.Entry(visibility=False)